frigate/object_processing.py

```diff
@@ -1032,18 +1033,17 @@ def get_best(self, camera, label):
         else:
             return {}
 
-    def get_current_frame(self, camera, draw_options={}):
+    def get_current_frame(self, camera, draw_options={}) -> tuple[Mat, float]:
```

Comment:
This function is used in other places and this change will break those usages

Reply:
> This function is used in other places and this change will break those usages

sure?

Reply:
Yes

https://github.com/blakeblackshear/frigate/blob/7a2d09dc3515f854b21376e08cd35037c8248cc3/frigate/http.py#L888

https://github.com/blakeblackshear/frigate/blob/7a2d09dc3515f854b21376e08cd35037c8248cc3/frigate/http.py#L1618